Scroller/EditableWrapper.js

```diff
@@ -374,6 +400,32 @@ const EditableWrapper = (props) => {
 		}
 	}, [removeItem, editable.removeItemFuncRef]);
 
+	useEffect(() => {
+		// addEventListener to moveItems while scrolled
+		const scrollContentNode = scrollContentRef.current;
+		const scrollContentCenter = scrollContentNode.getBoundingClientRect().width / 2;
+
+		const handleMoveItemsByScroll = () => {
+			const {centeredOffset, itemWidth, mouseClientX, selectedItem} = mutableRef.current;
+			if (selectedItem) {
+				const toIndex = Math.floor((mouseClientX + scrollContentNode.scrollLeft - centeredOffset) / itemWidth);
+				mutableRef.current.lastInputType = 'scroll';
+				if (mouseClientX > scrollContentCenter) {
+					moveItems(clamp(0, dataSize - 1, toIndex + 1));
```

Comment:
Do we need this clamp? Since we are bailing the outranged index in `moveItems` already.